draft-ietf-rtcweb-jsep.xml

```diff
@@ -1785,17 +1785,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           to support one of these profiles. For instance, an endpoint
           might generate "RTP/AVP" but supply "a=fingerprint" and
           "a=rtcp-fb" attributes, indicating its willingness to support
-          "(UDP,TCP)/TLS/RTP/SAVPF". In order to simplify compatibility
+          "UDP/TLS/RTP/SAVPF" or "TCP/TLS/RTP/SAVPF". In order to simplify compatibility
           with such endpoints, JSEP implementations MUST follow the
           following rules when processing the media m= sections in a
           received offer:</t>
 
           <t>
             <list style="symbols">
-              <t>Any profile in the offer matching the following patterns
-              MUST be accepted: "RTP/[S]AVP[F]" and
-              "(UDP|TCP)/TLS/RTP/SAVP[F]".
-              </t>
+              <t>Any profile in the offer matching one of the following
+              MUST be accepted:
+              "RTP/AVP", "RTP/AVPF", "RTP/SAVP", "RTP/SAVPF",
+              "UDP/TLS/RTP/SAVP", "UDP/TLS/RTP/SAVPF",
+              "TCP/TLS/RTP/SAVP", or "TCP/TLS/RTP/SAVPF".</t>
```

Comment:
I note that this excludes:
UDP/TLS/RTP/AVP[F]

This is actually a normative change, though I guess a safe one.